netbench/netbench/src/multiplex.rs

```diff
@@ -0,0 +1,441 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{connection::Owner, Result};
+use bytes::Buf;
+use core::{
+    pin::Pin,
+    task::{Context, Poll},
+};
+use futures::ready;
+use std::collections::{hash_map::Entry, HashMap, HashSet, VecDeque};
+use tokio::io::{AsyncRead, AsyncWrite};
+
+mod buffer;
+mod frame;
+mod stream;
+
+use buffer::{ReadBuffer, WriteBuffer};
+use frame::Frame;
+use stream::{ReceiveStream, SendStream, Stream};
+
+const DEFUALT_STREAM_WINDOW: u64 = 256000;
```

Comment:
```suggestion
const DEFAULT_STREAM_WINDOW: u64 = 256000;
```
probably should rebase to have the Typo check run